openam-ui/openam-ui-ria/src/main/resources/locales/ja/translation.json

```
@@ -0,0 +1,1292 @@
+                    "title": "__subSchema__ の追加"
+                },
+                "confirmDeleteSelected": "この 設定 を削除してよろしいですか?",
+                "callToAction": "セカンダリ設定が見つかりません。"
+            }
+        },
+        "realms": {
+            "title": "レルム",
+            "description" : "対象および設定データを整理するためにレルムを使用します。各レルム内では、データストア、管理権限、認証連鎖、認可ポリシー、およびその他のレルム固有の設定を行うことができます。",
+            "newRealm": "新規レルム",
+            "grid": {
+                "header": ["レルム名", "状態", "パス", "レルム / DNS エイリアス"]
+            },
+            "parentRealmCannotDeleted": "親レルムを削除することはできません。最初にサブレルムを削除してください。",
+            "edit": {
+                "title": "レルム"
+            },
+            "warningDialog": {
+                "title": "\"__realmName__\" を削除しますか?",
+                "activateMessage": "これはアクティブなレルムです。削除した場合は、すべての設定データが失われます。設定データを保持してログインを防ぎたい場合は、レルムを非アクティブに設定することができます。<br>このレルムを削除しますか?",
+                "deactivateMessage": "これは、非アクティブなレルムです。現在誰もログインすることはできません。削除した場合は、すべての設定データが失われます。<br>このレルムを削除してもよろしいですか?"
+            },
+            "realmStatusPopover": {
+                "title": "レルムの状態",
+                "content": "レルムがアクティブではない場合、すべての OpenAM サービスはこのレルム内のすべてのユーザーに対して使用できなくなります。"
+            },
+            "realmAliasesPopover": {
+                "title": "レルム / DNS エイリアス",
+                "content": "認証ユーザーインターフェースがリクエストが受信すると、ユーザーが認証されるべきレルムを見つけるため、 OpenAM はすべてのレルムに対してこの属性を検索します。 OpenAM は一致するレルムを1つだけを見つける必要しかないので、複数のレルムに重複したエントリを入れてはいけません。さらに OpenAM が一致するレルムを見つけられない場合、ユーザーにはエラー画面が表示されます。"
+            },
+            "realmNameValidationError" : "レルム名に文字 $, &, +, \\, \", カンマ, /, :, ;, =, ?, @, スペース, #, %, <, > は許可されていません。"
+        },
+        "scripts": {
+            "edit": {
+                "title": "新規スクリプト",
+                "language": "言語",
+                "script": "スクリプト",
+                "scriptType": "スクリプトタイプ",
+                "validation": {
+                    "line": "行",
+                    "column": "列",
+                    "message": "メッセージ",
+                    "errors": "このスクリプトにはエラーがあります:",
+                    "noErrors": "エラーは見つかりませんでした."
+                },
+                "dialog": {
+                    "title":  "スクリプトタイムの変更",
+                    "warning": "スクリプトタイプを変更すると、未保存のデータが失われます。"
+                }
+            },
+            "list": {
+                "title": "スクリプト一覧",
+                "addNew": "新規スクリプト",
+                "grid": ["名前", "スクリプトタイプ", "言語", "説明"],
+                "confirmDeleteText": "選択したスクリプトを削除してもよろしいですか?"
+            }
+        },
+        "configuration" : {
+            "authentication" : {
+                "description": "グローバルな認証機能のデフォルトプロパティを設定します。",
+                "core": {
+                    "coreAttributes": "コア属性",
+                    "title": "デフォルト認証レルム",
+                    "description": "新しいレルムを作成するときに適用するデフォルトの認証プロパティを設定します。"
+                },
+                "modules": {
+                    "title": "デフォルト認証モジュール",
```

`認証モジュールのデフォルト設定`

---

I think that it is a reasonable proposal, thank you.
```suggestion
                    "title": "認証モジュールのデフォルト設定",
```